homeassistant/components/sma_manager/config_flow.py

```diff
@@ -0,0 +1,96 @@
+"""SMA Manager Config Flow for UI configuration."""
+
+# Built-In Imports
+import logging
+import re
+
+# 3rd-Party Imports
+from sma_manager_api import SMA
+from voluptuous import Required, Schema
+
+# Home Assistant Imports
+from homeassistant import config_entries
+
+# Local Imports
+from .const import CONF_HOST, CONF_NAME, CONF_PORT, CONF_REFRESH_INTERVAL, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Enables the Integration to be configured in the UI."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_PUSH
+
+    def __init__(self):
+        """Init Config Flow object with data default values."""
+        self._data = {
+            CONF_NAME: "SMA Manager",
+            CONF_HOST: "239.12.255.254",
```

Comment:
What's up with this default IP address, how is it helpful?

Reply:
The device by default broadcasts UTP on that ip. It sends raw bytes through that IP and with sockets we just listen for that data.

Reply:
Ah, so the device relies on UDP multicast? In that case, why is this configurable?